### DISCUSSION TITLE
Mention Action Reference params on web site / docs

### DISCUSSION THREAD

**Discussion by erf** [ghostty-org/ghostty#3404] [Feature Requests, Ideas]
I think the params for the various [actions references](https://ghostty.org/docs/config/keybind/reference) you can bind to should be mentioned in the docs.

E.g. for [new_split](https://ghostty.org/docs/config/keybind/reference#new_split) the possible direction params are not mentioned and for [inspector](https://ghostty.org/docs/config/keybind/reference#inspector) the params like `toggle` are only mentioned in the text description. 

I think this could be improved by listing all the various parameters in a nice way perhaps below the description.

I mentioned this first in the [Discord help post here](https://discord.com/channels/1005603569187160125/1321982014634594314) , Kat also mentioned it [here](https://discord.com/channels/1005603569187160125/1320161293524795412/1322136346558595072).

**Comment by 00-kat** [comment]
cc @pluiedev does your overhaul of config help generation also address this?

**Reply by pluiedev** [reply]
I haven't touched actions yet but I intend to

**Comment by erf** [comment]
Note some improvements to this was done in https://github.com/ghostty-org/ghostty/pull/5344 and could probably be used for additional config settings.
